dateutil/tz/tz.py

```diff
@@ -1531,4 +1531,21 @@ def __enter__(self):
     def __exit__(*args, **kwargs):
         pass
 
+def resolve_imaginary(dt):
+    import pdb;pdb.set_trace()
```

Comment:
Is the Pdb needed outside of testing?

Reply:
Sorry I was finishing in a rush, should be for testing only, I can tidy it up and push it again.